HACKING.md

```diff
@@ -392,24 +355,43 @@ source files if `run-checks` complains about the format.
 You can run an individual test for a sub-package by changing into that 
 directory and:
 
-    go test -check.f $testname
+```
+go test -check.f $testname
+```
 
 If a test hangs, you can enable verbose mode:
 
-    go test -v -check.vv
+```
+go test -v -check.vv
+```
 
-(or -check.v for less verbose output).
+Or, try just `-check.v` for a less verbose output.
+
+> `snapd` before printing any text adjusts it to a dialect that
+corresponds to the locale of the hosting system. For instance, 
+texts for `en_US` and for `en_GB` not always look alike.
+Unfortunately, some tests don't take it into account (for the details 
+see [issue #1960131](https://bugs.launchpad.net/snapd/+bug/1960131)).
+A temporary workaround is to specify `LANG=C.UTF-8 ` in the command 
+line right before `go test`, if you experience similar issues.
```

Comment:
Just a suggestion, slightly more concise
```suggestion
> Before printing text, `snapd` translates it according to the 
system's locale. Currently, some unit tests don't take this into 
account (see [issue #1960131](https://bugs.launchpad.net/snapd/+bug/1960131)), which might
cause tests to fail on locales other than US English.
A temporary workaround is to set `LANG=C.UTF-8 ` when running `go test`.
```

Reply:
```suggestion
> Before printing text, `snapd` adjusts it according to the 
system's locale. Currently, some unit tests don't take this into 
account (see [issue #1960131](https://bugs.launchpad.net/snapd/+bug/1960131)), which might
cause tests to fail on locales other than US English.
A temporary workaround is to set `LANG=C.UTF-8 ` when running `go test`.
```

@MiguelPires, I don't think translate is the right word here, as it makes really small adjustments, far from real translation. Therefore, I recommend to use verb "adjust" instead of "translates". 
Would it be okay with you?

Reply:
That's only the case if your locale is something like en_GB, in which case the adjustments are small (e.g., "color" -> "colour"). If your locale is an entirely different language, then it will translate it (depending on available translations).